packages/forms-web-app/__tests__/unit/controllers/projects/examination-timetable/controller.compoent.test.js

```diff
@@ -0,0 +1,323 @@
+const {
+	getExaminationTimetable,
+	postExaminationTimetable
+} = require('../../../../../src/controllers/projects/examination-timetable/controller');
+
+const { getAppData } = require('../../../../../src/services/application.service');
+const { getTimetables } = require('../../../../../src/lib/application-api-wrapper');
+
+jest.mock('../../../../../src/services/application.service', () => ({
+	getAppData: jest.fn()
+}));
+jest.mock('../../../../../src/lib/application-api-wrapper', () => ({
+	getTimetables: jest.fn()
+}));
+
+const { mockReq, mockRes, mockResponse } = require('../../../mocks');
+
+const expectUnhandledException = (res) => {
+	expect(res.render).toHaveBeenCalledWith('error/unhandled-exception');
+	expect(res.status).toHaveBeenLastCalledWith(500);
+};
+
+const expectNotFound = (res) => {
+	expect(res.render).toHaveBeenCalledWith('error/not-found');
+	expect(res.status).toHaveBeenLastCalledWith(404);
+};
+
+describe('controllers/projects/examination-timetable/controller', () => {
+	let req;
+	let res;
+
+	describe('examination-timetable controllers', () => {
+		beforeEach(() => {
+			jest.useFakeTimers().setSystemTime(new Date('2023-01-02'));
+		});
+		describe('#getExaminationTimetable', () => {
+			beforeEach(() => {
+				req = {
+					...mockReq(),
+					params: {
+						case_ref: 'mock case ref'
+					},
+					session: {}
+				};
+				res = mockRes();
+				jest.useFakeTimers().setSystemTime(new Date('2023-01-02'));
```

Comment:
duplicate timer set

maybe put the date in a const called dateOverride ? or something better